### DISCUSSION TITLE
License vs Copyright for manifest

### DISCUSSION THREAD

**Discussion by superusercode** [microsoft/winget-pkgs#12254] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/60588434/117528217-0eece880-af9f-11eb-8a7b-d5f424f2553e.png)

Currently the application license field in the YamlCreate still includes that shows `Copyright (c) Microsoft Corporation`. Is it intended for a manifest to just reuse that line again for the copyright field or should `Proprietary` go in there instead? When updating the valve manifest (https://github.com/microsoft/winget-pkgs/blob/master/manifests/v/Valve/Steam/2.10.91.91/Valve.Steam.locale.en-US.yaml#L12-L14) I used it twice.

**Comment by denelon** [comment, marked as answer]
Before we had a `Copyright` or `CopyrightUrl` field, many packages just put the copyright in the `License` field. For open source projects, I like to try and find a copyright if one is available, and otherwise I use the license file URL for the `LicenseUrl`.
